src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+* {
   margin: 0;
 }
+
+html {
+  --header-height: 60px;
+  --link-color: #00acdc;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  width: 100%;
+  max-width: 1920px;
+}
+
+.container {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  max-width: 1200px;
+}
```

Comment:
```suggestion

```

Reply:
is it for .header or .container suggestion?